_config/config.yml

```diff
@@ -8,3 +8,9 @@ File:
 Director:
   rules:
     'assets': 'SecureFileController'
+SecureAssets:
+  Defaults:
+    Permission: 'Anyone'
+    Groups:
+      - 'Administrators'
+      - 'Content Authors'
```

Comment:
You'll need to use the group code here, not the group titles.

E.g. `content-authors`, `administrators` are the default codes.